src/TextLink/__examples__/ACCESSIBILITY.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import * as React from "react";
+
+import TextLink from "../index";
+import Heading from "../../Heading";
+import Stack from "../../Stack";
+import Text from "../../Text";
+import * as Icons from "../../icons";
+
+export default {
+  Example: () => (
+    <Stack direction="column">
+      <Stack shrink direction="column" spacing="condensed">
+        <Heading type="title3" as="h3">
+          Tab index
+        </Heading>
+        <Text>
+          The <code>tabIndex</code> prop makes the given element focusable in keyboard navigation.
```

Comment:
@CollierCZ This is not exactly true. It's true with positive numbers but it can also be used to hide an element from the accessibility tree. This prop in general should be used only for edge-cases for example when you are rendering TextLink in to non actionable elements like `div` or you want to hide that element from accessibility tree. Could you mention this there as now it can be seen we promote that as preferred solution to make something focusable? :)

Reply:
Better now?